karrot/groups/models.py

```diff
@@ -62,9 +54,12 @@ def annotate_yesterdays_member_count(self):
             )
         )
 
+# to avoid circular imports, managers has to be imported after GroupQuerySet
+from karrot.groups import managers
 
 class Group(BaseModel, LocationModel, ConversationMixin, DirtyFieldsMixin):
-    objects = GroupQuerySet.as_manager()
+    # objects = GroupQuerySet.as_manager()
+    objects = managers.GroupManager()
```

Comment:
Have a look at https://docs.djangoproject.com/en/3.0/topics/db/managers/#from-queryset to avoid having to duplicate queryset methods in the custom manager.

Reply:
The changes I made in my master branch should show up I here automatically, right?
I have currently no time to test this (because I am going to paint our fairteiler now.)